modules/RTC/DataChannels.js

```diff
@@ -13,6 +13,7 @@ function DataChannels(peerConnection, emitter) {
     peerConnection.ondatachannel = this.onDataChannel.bind(this);
     this.eventEmitter = emitter;
 
+    this._lastNEndpoints = null;
```

Comment:
I don't think this is the right place for this. The data channel is just a message bus, the last N endpoints are related to the RTC connection and IMHO it feels much better in RTC.js.